Caltech-IPAC/firefly
# IRSA-1605: Readout working with HiPS

- pixel now show angular size of the HiPS pixel for the displayed level
 - options cleaned up to work better with HiPS
 - some code refactoring to make `MouseReadoutUIUtil` for reused among readout code
 - general cleanup of readout

_to test:_
 - load an image and it should work as before.
-  load image and you should see the image pixel readout change with hips level
- under readout options: HiPS should not have image point as an option
